docs/ADDRESS_LOOKUP.md

```diff
@@ -11,7 +11,63 @@
 - If you're integrating with Drop-in:
     - Implement the mandatory `onAddressLookupQueryChanged(query: String)` callback and optional `onAddressLookupCompletion(lookupAddress: LookupAddress)` callback.
     - Pass the result of these actions by using `AddressLookupDropInServiceResult` class.
+      ```kotlin
+      override fun onAddressLookupQueryChanged(query: String) {
```

Comment:
Can we mention that these code samples are inside their implementation of `DropInService`? People using sessions might not have seen this class at all